modules/plugins/git/gitsigns/config.nix

```diff
@@ -69,7 +69,7 @@ in {
         };
 
         binds.whichKey.register = pushDownDefault {
-          "<leader>g" = "+Gitsigns";
```

Comment:
This is a breaking change, we might need to warn the user about the changes but even if we don't, this *should* be in the release notes in the breaking changes section.

Reply:
Good catch, I left a note in the breaking changes section.